app/src/main/java/com/safaorhan/reunion/FirestoreHelper.java

```diff
@@ -102,7 +103,7 @@ public static void sendMessage(final String messageText, final DocumentReference
                     @Override
                     public void onSuccess(DocumentReference documentReference) {
                         conversationRef
-                                .update("lastMessage", documentReference)
+                                .update("lastMessage", documentReference, "conversationTime", Timestamp.now())
```

Comment:
I think you don't need to send conversationTime since it's annotated as @ServerTimestamp.

It should get updated everytime you do an update.